waiter/src/waiter/cors.clj

```diff
@@ -44,16 +45,16 @@
   "Preflight request handling middleware.
   This middleware needs to precede any authentication middleware since CORS preflight
   requests do not support authentication."
-  [handler cors-validator max-age]
-  (fn wrap-cors-preflight-fn [request]
+  [handler cors-validator max-age kv-store token-defaults waiter-hostnames]
+  (fn wrap-cors-preflight-fn [{:keys [headers] :as request}]
     (if (preflight-request? request)
```

Comment:
We should populate `access-control-allow-methods` from the configired `methods` on the token `allowed-cors`?